pommerman/agents/player_agent_blocking.py

```diff
@@ -0,0 +1,37 @@
+from time import time
+import click
+
+from . import BaseAgent
+from .. import characters
+from .. import constants
+
+
+class PlayerAgentBlocking(BaseAgent):
+    """Block for keyboard input."""
+
+    def __init__(self, character=characters.Bomber, agent_control='arrows'):
+        super(PlayerAgentBlocking, self).__init__(character)
+        self._character.agent_image = 'agent-player'
+        self.agent_control=agent_control
+
+    def act(self, obs, action_space):
+        key = click.getchar()
```

Comment:
Does this block? Is that what the mechanism is?

Reply:
yes.